lib/flutter_login.dart

```diff
@@ -220,6 +220,8 @@ class FlutterLogin extends StatefulWidget {
       @required this.onLogin,
       @required this.onRecoverPassword,
       @required this.onConfirmRecover,
+      @required this.onConfirmSignup,
+      @required this.onResendCode,
```

Comment:
Right now the new cards are required. I don't think is a good solution since this will force users to use the new cards, even if they don't need them. This is the same for the code currently in confirmation-card branch. I will change that code and update that branch. After that, this pull-request also needs to be updated to make the new cards optional.

Reply:
@doc-rj-celltrak i have made onConfirmRecover optional. Is it possible to merge changes and make onConfirmSignup and onResendCode optional? Thank you in advance

Reply:
There are some more changes required as tests are failing right now on confirmation-card branch. I will update you when this is fixed.